ODROID-C2.md

```diff
@@ -0,0 +1,15 @@
+## Additional Packages
+
+* X.org video driver package: xf86-video-fbdev
+* Mali EGL and GLES drivers (currently framebuffer only): odroid-c2-libgl-fb
+* wiringPi modified for the ODROID-C1/C2: wiringc1
+
+## Display
+
+Configuring the resolution of the screen on the HDMI port is done through two items
+
+* /boot/boot.ini
+* /usr/bin/amlogic.sh (called by amlogic.service)
+
+The configurations are very similar to the [[ODROID-C1]] in this regard. At this time, the settings are taken from ``/proc/cmdline`` and as such require a reboot for changes to take effect. If you are going to use the console or X, enable ``amlogic.service`` to ensure that your display settings are properly applied early in the boot process.
```

Comment:
That does seem to be the case. It seems I didn't check the latest tarball vs my initial installation.